javascript/chubbyjs/package.json

```diff
@@ -3,7 +3,7 @@
     "@chubbyjs/chubbyjs-framework": "~1.1.1",
     "@chubbyjs/chubbyjs-framework-router-path-to-regexp": "~1.0.1",
     "@chubbyjs/chubbyjs-http-message": "~1.1.1",
-    "@chubbyjs/chubbyjs-node-psr-http-message-bridge": "~1.2.1",
+    "@chubbyjs/chubbyjs-node-psr-http-message-bridge": "~1.3.0",
     "@types/node": "14"
```

Comment:
```suggestion
    "@chubbyjs/chubbyjs-framework-router-path-to-regexp": "*",
    "@chubbyjs/chubbyjs-http-message": "*",
    "@chubbyjs/chubbyjs-node-psr-http-message-bridge": "*",
    "@types/node": "*"
```

Reply:
can we use something to map `@types/node` to major node version in use ? in could be weird to specify `"@types/node": "12"` when using `node` **14**

Reply:
"@types/node": "*" should be ok, the other changes means if there is a major upgrade changes are things will break,